priv/worker.py

```diff
@@ -232,7 +247,7 @@ def process_response(response):
       envs.env_close(instance_id)
 
     instance_id = envs.create(enviroment)
-    data = json.dumps({"instance" : instance_id})
+    data = json.dumps({"instance" : instance_id},cls=NDArrayEncoder)
```

Comment:
Picky comment. Please space this line e.g. `, cls = NDArrayEncoder`.

Reply:
Sure